output/typescript/types.ts

```diff
@@ -6022,9 +6022,15 @@ export interface MappingSourceField {
 
 export type MappingSourceFieldMode = 'disabled' | 'stored' | 'synthetic'
 
+export interface MappingSparseVectorIndexOptions {
+  prune?: boolean
+  pruning_config?: QueryDslTokenPruningConfig
```

Comment:
Why do we have both `QueryDslTokenPruningConfig` and `TokenPruningConfig`? Can we consolidate?

Reply:
I don't know - this file is one of the ones automatically generated when running `make contrib`

Reply:
Hmmm... I'm pretty sure it added the `QueryDsl*` prefix as it imports the TokenPruningConfig from there ('@_types/query_dsl/TokenPruningConfig')...  @kderusso - WDYT about moving the TokenPruningConfig to a more common place? I _don't think_ it would break backwards compatibility (as the final output structure would be the same)...?